obspy/imaging/maps.py

```diff
@@ -144,7 +151,6 @@ def plot_basemap(lons, lats, size, color, labels=None,
     if show_colorbar:
         map_ax = fig.add_axes([ax_x0, 0.13, ax_width, 0.77])
         cm_ax = fig.add_axes([ax_x0, 0.05, ax_width, 0.05])
-        plt.sca(map_ax)
```

Comment:
I remember looking for these and _not_ doing this one for some reason, but I don't seem to recall what it was.